shared.js

```diff
@@ -233,11 +233,19 @@ function BlockUser(user, user_id, headers, reason, attempt=1) {
 const blockableAffiliateLabels = new Set(["AutomatedLabel"]);
 const blockableVerifiedTypes = new Set(["Business"]);
 export function BlockBlueVerified(user, headers) {
+	if (!user?.rest_id) {
```

Comment:
This verifies that there is a `user` object and has the critical `rest_id`
Other checks don't need to check if `user` is defined after this